### DISCUSSION TITLE
problem with x-for on this release 3.4.2

### DISCUSSION THREAD

**Discussion by iketeshima** [alpinejs/alpine#2161] [7. Bugs (archived)]
In this last update, **x-for** parameter is duplicating list of records

```
<ul x-data="{ colors: ['Red', 'Orange', 'Yellow'] }">
    <template x-for="color in colors">
        <li x-text="color"></li>
    </template>
</ul>
```

Results:

Red
Orange
Yellow
Red
Orange
Yellow

**Comment by SimoTod** [comment]
Can you post an example? It works correctly here: https://codepen.io/SimoTod/pen/abwxWJO
